driver-core/src/main/java/com/datastax/driver/core/ProtocolOptions.java

```diff
@@ -231,4 +232,23 @@ public AuthProvider getAuthProvider() {
         return authProvider;
     }
 
+    /**
+     * Allows client to connect to server using latest development protocol version,
+     * which is currently in beta.
+     *
+     * @param useBeta
+     * @return this {@code ProtocolOptions} object.
+     */
+    public ProtocolOptions setAllowBetaProtocolVersions(boolean useBeta) {
```

Comment:
I don't think we can change this value once the `Cluster` is created, because we would have to remove the `Frame.BetaVersionFlagAdder` handler from all channels. So I's suggest we make this field `final` and remove this setter.